travis_config.sh

```diff
@@ -76,9 +76,8 @@ if [ -n "$IS_OSX" ]; then
         if [ -n "$REGENERATE" ]; then
             echo "Regenerating custom ffmpeg formula"
             # Bottle block syntax: https://docs.brew.sh/Bottles#bottle-dsl-domain-specific-language
-            # FfmpegAT4 is a class in ffmpeg@4 formula
             perl -wpe 'BEGIN {our ($found_blank, $bottle_block);}
-                if (/(^class )(FfmpegAT4)(\s.*)/) {$_=$1."FfmpegOpencv".$3."\n"; next;}
```

Comment:
You can leave `FfmpegOpencv` here, because the previous logic was something like `$2.Opencv` what meant `FfmpegOpencv`, but wasn’t obvious.

link to the previous logic (before my changes), notice `$2` here: https://github.com/opencv/opencv-python/commit/602a8be9e21df225a32827a621f15d7be0e5eebd#diff-7d5a84b3617f8aff01b2fc673bb198946e3632d90592fd61d3e3e24ddd6dbf70L74

Reply:
Yes, it works, tried in my fork.

Log: https://github.com/opencv/opencv-python/actions/runs/3694906850/jobs/6256653970